src/frontend/main.go

```diff
@@ -114,7 +114,7 @@ func InitMeterProvider() (*controller.Controller, error) {
 		return nil, err
 	}
 
-	controller := controller.New(
+	pusher := controller.New(
```

Comment:
This change is to avoid overriding controller which package that is being imported.